keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -120,8 +124,10 @@ data class ClusterSpec(
     capacity: Capacity?,
     dependencies: ClusterDependencies?,
     health: HealthSpec?,
+    @JsonInclude(NON_EMPTY)
     scaling: Scaling?,
     tags: Map<String, String>?,
+    @JsonInclude(NON_EMPTY)
```

Comment:
Does this just mean the property is omitted on serialization if it's `null`?

Reply:
Or if it's an empty collection